recipes/trompeloeil/all/conandata.yml

```diff
@@ -0,0 +1,16 @@
+sources:
+  "35":
+    url: https://github.com/rollbear/trompeloeil/archive/v35.tar.gz
+    sha256: 5F3D8366876C199522F30D7B9C5CE6E489119C126F6E743FFF44FA8539599EC8
+  "36":
+    url: https://github.com/rollbear/trompeloeil/archive/v36.tar.gz
+    sha256: B2F5DB3C9A892B80ED5DC1E09E9528D17F974F02BF0A508CD2A892A6969F0348
+  "37":
+    url: https://github.com/rollbear/trompeloeil/archive/v37.tar.gz
+    sha256: 770555FA8F550B91F239FA4987FABF49C294EAFC9BD73EB081D67D1D58CE51E8
+  "38":
+    url: https://github.com/rollbear/trompeloeil/archive/v38.tar.gz
+    sha256: 7E452AD0FA7CFD84A0AF7BDFE5CDFCC755C902A3B25ACA704865BDF14E538079
```

Comment:
Do you need all those versions yourself? If not, only add the latest version please

Reply:
I added the versions that "are available" on conan center right now: https://conan.io/center/trompeloeil

Reply:
Should I remove all other versions and only keep 39? 

I mean the work is done, and its a header only, so no binaries involved to be stored?

Reply:
Same question to you as in the beginning: Are you using all of those versions? If yes, keep them, if not, remove all you don't need except the newest.

Reply:
done